src/NuGet.Core/NuGet.Protocol.Core.Types/Resources/DownloadResource.cs

```diff
@@ -17,6 +17,7 @@ public abstract class DownloadResource : INuGetResource
         public abstract Task<DownloadResourceResult> GetDownloadResourceResultAsync(
             PackageIdentity identity,
             ISettings settings,
+            NuGet.Logging.ILogger logger,
```

Comment:
Using fully qualified type name here because there's also a conflicting `NuGet.ILogger` in the NuGet.Core.nupkg